.github/workflows/scripts/ti_build/compiler.py

```diff
@@ -25,7 +25,7 @@ def setup_clang(as_compiler=True) -> None:
     """
     u = platform.uname()
     if u.system in ("Linux", "Darwin"):
-        for v in ("", "-14", "-13", "-12", "-11", "-10"):
+        for v in ("", "-16", "-15", "-14", "-13", "-12", "-11", "-10"):
```

Comment:
I can't blindly allow higher versions of Clang because they do cause problems, please revert this and put it into a separate PR and then run tests there.